### PR TITLE
feat(oui-select-picker): add transclusion for label

### PR DESCRIPTION
**Pull request by JeremyDec**
Signed-off-by: Jérémy De-Cesare <jeremy.de-cesare@corp.ovh.com>

Backport on V3 can be done after the acceptation of this changes.
Some changes from prettier has been done

### CODE REVIEW

File: packages/oui-select-picker/src/select-picker.controller.js [hunk 1]
```diff
@@ -37,23 +42,34 @@ export default class SelectPickerController {
...
+        return (
+            this.$transclude.isSlotFilled("descriptionSlot") || this.description
+        );
     }
 
     hasFooter () {
         return this.$transclude.isSlotFilled("footerSlot") || this.footer;
     }
 
+    hasLabel () {
```
Comment (JeremyDec) [inline packages/oui-select-picker/src/select-picker.controller.js]: Only this has been added

File: packages/oui-select-picker/src/select-picker.html [hunk 2]
```diff
@@ -7,76 +9,100 @@
...
+            />
+            <span ng-class=":: $ctrl.picture" ng-if="!$ctrl.isImgPath"> </span>
         </span>
 
         <!-- Label -->
-        <span class="oui-select-picker__label">
-            <span id="{{:: $ctrl.id }}Label"
-                ng-bind="$ctrl.label">
-            </span>
+        <span
```
Comment (JeremyDec) [inline packages/oui-select-picker/src/select-picker.html]: Only this part has changed

**Comment by AxelPeter** [comment]
@JeremyDec can you revert the changes from prettier and only add the needed changes?

File: packages/oui-select-picker/src/select-picker.html [hunk 3]
```diff
@@ -26,7 +26,9 @@
         </span>
 
         <!-- Label -->
-        <span class="oui-select-picker__label">
+        <span class="oui-select-picker__label"
+              id="{{:: $ctrl.id }}Label"
+              ng-transclude="labelSlot">
             <span id="{{:: $ctrl.id }}Label"
```
Comment (AxelPeter) [inline packages/oui-select-picker/src/select-picker.html]: Careful, you'll get a double ID here. You could remove the one on this `span` ;)

**Pull request merged**